Vates/VatesSimpleGui/ViewWidgets/inc/MantidVatesSimpleGuiViewWidgets/MdViewerWidget.h

```diff
@@ -1,6 +1,9 @@
 #ifndef MDVIEWERWIDGET_H_
 #define MDVIEWERWIDGET_H_
 
+#ifdef ERROR
+#undef ERROR
+#endif
```

Comment:
@martyngigg Any idea where the `ERROR` macro is coming from? Without this, windows won't build.